Sources/MapboxNavigation/DestinationOptions.swift

```diff
@@ -0,0 +1,36 @@
+import CoreLocation
+import MapboxDirections
+
+// :nodoc:
+public struct DestinationOptions {
+    
+    // :nodoc:
+    public var primaryText: String?
+    
+    // :nodoc:
+    public var secondaryText: String?
```

Comment:
Should these properties accept an AttributedString or AttributedStringProtocol, in case an application needs to apply some inline text formatting? `UILabel.attributedText` can be set to an NSAttributedString; it’s possible to convert between the two types.

Since these properties are undocumented, support for attributed text can be tackled separately.

Reply:
Good idea, both `AttributedString` and `AttributedStringProtocol` protocols are only available since iOS 15.x though. I'll see what we can do here though.